From 3981482ba429e650c0b72ec3cfc1046d7ee50206 Mon Sep 17 00:00:00 2001
From: Joonas Lahtinen <joonas.lahtinen@linux.intel.com>
Date: Tue, 9 Feb 2021 15:52:48 +0200
Subject: [PATCH 2/2] nightly.conf: Move "drm-intel-gt-next" right after
 "drm-intel-next"

Fix the ordering to localize merge conflicts from the two branches.

Suggested-by: Jani Nikula <jani.nikula@intel.com>
Signed-off-by: Joonas Lahtinen <joonas.lahtinen@linux.intel.com>
---
 nightly.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nightly.conf b/nightly.conf
index 2d7c6c7d1eaa..5477179b0d4c 100644
--- a/nightly.conf
+++ b/nightly.conf
@@ -85,12 +85,12 @@ drm_tip_config=(
 
 	"drm-misc		drm-misc-next"
 	"drm-intel		drm-intel-next"
+	"drm-intel		drm-intel-gt-next"
 	"drm-amd		drm-amd-next"
 
 	"sound-upstream		for-linus"
 	"sound-upstream		for-next"
 	"drm-intel		topic/core-for-CI"
-	"drm-intel		drm-intel-gt-next"
 	"drm		topic/iomem-mmap-vs-gup"
 	"drm		topic/nouveau-ampere-modeset"
 	"drm-intel		topic/adl-s-enabling"
-- 
2.29.2

